packages/cashier/src/Sass/cashier.scss

```diff
@@ -1396,3 +1396,55 @@ div.payment-agent-transfer {
     }
 }
 /* stylelint-enable plugin/selector-bem-pattern */
+
+.cashier-default {
+    &-header {
+        text-align: center;
+    }
```

Comment:
instead of using text-align you can use "align" on text component

Reply:
@mehrzad-fs that doesn't work here, I wanted to place the text in the center of other `div`